6.2.the_2000_run.py

```diff
@@ -0,0 +1,24 @@
+import time
+
+
+def timer(f, *args, **kwargs):
+    """
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names
Try to avoid i and j as well if there is a better name available.

Reply:
Done